pillar/edx/init.sls

```diff
@@ -47,3 +47,10 @@ schedule:
     args:
       - etl.mitx
   {% endif %}
+  {% if 'mitx-' in environment %}
```

Comment:
To be more explicit about this we can filter on `business_unit == residential`

Reply:
Sure, but feels as if we're more consistent with the environment grain than the business_unit, but I'll set it as much